{{cookiecutter.project_slug}}/compose/local/docs/Dockerfile

```
@@ -0,0 +1,30 @@
+FROM python:3.8-slim-buster
+
+ENV PYTHONUNBUFFERED 1
+ENV PYTHONDONTWRITEBYTECODE 1
+
+RUN apt-get update \
+    # dependencies for building Python packages
+    && apt-get install -y build-essential \
+    # psycopg2 dependencies
+    && apt-get install -y libpq-dev \
+    # Translations dependencies
+    && apt-get install -y gettext \
+    # Uncomment below to enable Sphinx output to latex and pdf
+    # && apt-get install -y texlive-latex-recommended \
+    && apt-get install -y texlive-fonts-recommended \
+    && apt-get install -y texlive-latex-extra \
+    && apt-get install -y latexmk \
+    # cleaning up unused files
+    && apt-get purge -y --auto-remove -o APT::AutoRemove::RecommendsImportant=false \
+    && rm -rf /var/lib/apt/lists/*
+
+# Requirements are installed here to ensure they will be cached.
+COPY ./requirements /requirements
+# All imports needed for autodoc.
+RUN pip install -r /requirements/local.txt
+RUN pip install -r /requirements/production.txt
```

We could save a layer and specify both files at once, no? 

```suggestion
pip install -r /requirements/local.txt -r /requirements/production.txt
```